Makefile

```
@@ -0,0 +1,19 @@
+SHELL 		    := /bin/bash
+
+DIRS                := cnf-app-mac-operator nfv-example-cnf-index testpmd-container-app testpmd-lb-operator testpmd-operator trex-container-app trex-operator
```

I wonder if it would be better to split the list in apps, operators and catalog

```suggestion
APP_DIRS        := trex-container-app testpmd-container-app
OPERATOR_DIRS   := cnf-app-mac-operator testpmd-lb-operator testpmd-operator trex-operator
CATALOGS_DIR    := nfv-example-cnf-index
```

Then create targets to `build-apps`, `build-operators`, and `build-catalog`, and their corresponding `push-*` targets.
The reasoning for this suggestion is because a change in an app will generate an image. That image must be pushed to a repo, so that it later can be included in the operator, and once the operator produces an image it must be pushed as well so the catalog can use it.

---

I'm trying to get to the same state as it was before. let's do some changes like that later.

---

The previous state is quite similar to what I described above, the main difference is that it was per application, not all at once:

Change in an app --> build --> push --> update in the operator of the app (to point to the new image) --> build --> push --> update in the catalog to use the new operator image --> build --> push

hth

---

ok I have changed the order to respect that.